website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -184,11 +302,156 @@ Note that `value` must be defined as a string in YAML, because it will be compil
         value: "'2020-01-01'"
 ```
 
-## Ongoing discussions
+## Querying Your Metric
+If you're interested in querying your metric, you must have the [dbt_metrics package](https://github.com/dbt-labs/dbt_metrics) installed. Information on how to install packages in [your project can be found here](https://docs.getdbt.com/docs/building-a-dbt-project/package-management#how-do-i-add-a-package-to-my-project) - this can be used in conjunction with the code snippet below to ensure you can query metrics in your project. 
```

Comment:
```suggestion
If you're interested in querying your metric, you must have the [dbt_metrics package](https://github.com/dbt-labs/dbt_metrics) installed. 

You can review more information on how to [install packages in your project](https://docs.getdbt.com/docs/building-a-dbt-project/package-management#how-do-i-add-a-package-to-my-project) and use this together with the code snippet below to ensure you can query metrics in your project. 
```

Reply:
I will make a few edits here because the second sentence doesn't sound right for installing the package. Less using the code from the metrics package and more installation info.

Reply:
gotcha good call, maybe it's just linking the package but making sure they know they have to run dbt deps?